circuit/js/src/types.ts

```diff
@@ -10,3 +10,14 @@ export type RawInput<T> = {
     [P in keyof T]: ToRawInput<T[P]>;
 };
 
+export interface AxiomV2CircuitMetadataParams {
+    version: number;
+    numInstance: number[];
```

Comment:
this `numInstance` is different from the `numInstance` in `CircuitScaffold`, where the latter is `2 * resultsLen`. This is almost certainly going to cause confusion later. At the minimum we should leave some comments about this, or we rename one of them to something else.